src/browser/api/custom_frame.ts

```diff
@@ -0,0 +1,15 @@
+import { CustomFrameOptions } from '../../shapes';
+import { Window } from './window';
+const path = require('path');
+
+
+export async function create(id: number, options?: CustomFrameOptions, frameUrl?: string) {
+    const { uuid, name } = options;
+
+    const win = Window.create(id, {
```

Comment:
I'm deleting the entire file since it's redundant atm; 
For now, I'm replacing the mainWindow url with `default_frame.html` in `convertOptions.toCustomFrame` method.

Could you take a look and let me know if you're ok with it?